packages/syft/src/syft/types/errors.py

```diff
@@ -0,0 +1,51 @@
+# stdlib
+from typing import Literal
+
+# relative
+from ..service.user.user_roles import ServiceRole
+
+SyftErrorCodes = Literal[
+    "not-found",
+    "not-permitted",
+    "database-error",
+    "serde-serialization-error",
+    "serde-deserialization-error",
+    "stash-error",
+    "worker-invalid-pool",
+    "usercode-not-approved",
+    "usercode-bad-input-policy",
+    "usercode-bad-output-policy",
+]
+
+
+class SyftError(Exception):
```

Comment:
Perhaps `SyftException` to avoid shadowing SyftSuccess/SyftError?

Reply:
Hey @eelcovdw :)

This is not the final name. `SyftException` also exists... and so does `PySyftException`! By cleaning up, my aim is to take over `SyftException`.

Let's see :)